rackspace/autoscale/v1/launch/fixtures.go

```diff
@@ -0,0 +1,89 @@
+// +build fixtures
+
+package launch
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	th "github.com/rackspace/gophercloud/testhelper"
+	"github.com/rackspace/gophercloud/testhelper/client"
+)
+
+// LaunchGetBody contains the canned body of a launch.Get response.
+const LaunchGetBody = `
+{
+  "launchConfiguration": {
+    "args": {
+      "loadBalancers": [
+        {
+          "port": 443,
+          "loadBalancerId": 123456
+        }
+      ],
+      "server": {
+        "name": "server",
+        "imageRef": "40155f16-21d4-4ac1-ad65-c409d94b8c7c",
+        "key_name": "gophercloud",
+        "flavorRef": "performance1-4",
+        "user_data": "thequickbrownfoxjumpsoverthelazydog",
+        "config_drive": true,
+        "networks": [
+          {"uuid": "00000000-0000-0000-0000-000000000000"},
+          {"uuid": "11111111-1111-1111-1111-111111111111"}
+        ],
+        "metadata": {
+          "foo": "bar"
+        }
+      }
+    },
+    "type": "launch_server"
+  }
+}
+`
+
+// LaunchConfig is a Configuration struct corresponding to the result in LaunchGetBody.
+var LaunchConfig = Configuration{
+	Type: "launch_server",
+
+	Args: Args{
+		Server: map[string]interface{}{
+			"name":         "server",
+			"imageRef":     "40155f16-21d4-4ac1-ad65-c409d94b8c7c",
+			"key_name":     "gophercloud",
+			"flavorRef":    "performance1-4",
+			"user_data":    "thequickbrownfoxjumpsoverthelazydog",
+			"config_drive": true,
+			"networks": []map[string]string{
+				{"uuid": "00000000-0000-0000-0000-000000000000"},
+				{"uuid": "11111111-1111-1111-1111-111111111111"},
+			},
+			"metadata": map[string]string{
+				"foo": "bar",
+			},
+		},
+		LoadBalancers: []map[string]interface{}{
+			{
+				"port":           443,
+				"loadBalancerId": 123456,
+			},
+		},
+		DrainingTimeout: 0, // TODO: This is optional, is this right?
```

Comment:
I think you've got the right idea here. We want the fields to be as specific as possible, so for some things we have no choice but `map[string]interface{}` but we want that to be  a last resort. If you don't want the `DrainingTimeout` field to show up if it's empty, you can use the [`encoding/json`](https://golang.org/pkg/encoding/json/#Unmarshal) package (instead of the `mapstructure` package) to unmarshal the JSON into the `Configuration` and use `json:"draining_timeout,omitempty"` for the field's struct tag.